From bfbcc6d02f0621bb9ff5afc39862aac55ff305bf Mon Sep 17 00:00:00 2001
From: Simon Schricker <sschricker@suse.de>
Date: Wed, 15 May 2019 13:36:41 +0200
Subject: [PATCH] nvme-cli: add default IO-policy rule for NetApp E-Series

NetApp E-Series controller provide several paths to the same subsystem,
so we should be switching to 'round-robin' iopolicy to provide the best
performance.
---
 scripts/71-nvme-iopolicy-netapp-E-Series.rules | 2 ++
 1 file changed, 2 insertions(+)
 create mode 100644 scripts/71-nvme-iopolicy-netapp-E-Series.rules

diff --git a/scripts/71-nvme-iopolicy-netapp-E-Series.rules b/scripts/71-nvme-iopolicy-netapp-E-Series.rules
new file mode 100644
index 0000000..902c083
--- /dev/null
+++ b/scripts/71-nvme-iopolicy-netapp-E-Series.rules
@@ -0,0 +1,2 @@
+# Enable round-robin for NetApp E-Series
+ACTION=="add", SUBSYSTEM=="nvme-subsystem", ATTR{model}=="NetApp E-Series", ATTR{iopolicy}="round-robin"
-- 
2.16.4

